lib/src/upgrade_messages.dart

```diff
@@ -747,6 +747,12 @@ class UpgraderMessages {
       case 'te':
         message = 'విడుదల గమనికలు';
         break;
+      case 'tr':
+        message = 'Sürüm Notları';
+        break;
+      case 'ru':
```

Comment:
@kadambilal Thanks for this PR. Can you move the 'ru' up before 'te' to keep them in alphabetical order?